NEWS

```diff
@@ -11,6 +11,13 @@
 ### Features:
 ### Bugfixes:
 
+## 1.16.0-rc4 (February 21, 2024)
+### Bugfixes:
+#### UCT
+* Fixed Infiniband posting error when compiler optimizes code with memmove()
```

Comment:
afair you did not port that fix to `1.16`

Reply:
agree, but I am assuming that we will port it, else I would remove that line before merge.

Reply:
@yosefe, do we need to port #9692?

Reply:
created #9705, updated description of this PR